server/routes/loginAuth.js

```diff
@@ -65,6 +66,22 @@ router.get("/cookie", (req, res) => {
     });
 });
 
+// Returns the configured login services
+router.get("/services", (req, res) => {
+    const services = ["GOOGLE_CLIENT","FACEBOOK_CLIENT","TWITTER_CLIENT"]
+    const configuredServices = {};
+    for (const service of services) {
+        const providerName = service.split("_")[0].toLowerCase();
+        if (process.env[`${service}_SECRET`] && process.env[`${service}_ID`])
+            configuredServices[providerName] = true;
+    }
```

Comment:
Shouldn't this check occur when the server is initializing. That way we don't have to recompute this every time a request is made here and stop the server from launching if none of the social logins are configured